frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeList/CodeList.tsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Can change it to `CodeListPage` 👍  Then we can keep a consistent pattern for all future pages.

Reply:
But maybe it is better to do in a separate PR? As many files will be affected and will result in a large PR. 
This issue is not introduced by this PR so it can be done independantly 😊